src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEMethodSymbol.cs

```diff
@@ -106,6 +106,8 @@ private struct PackedFlags
             private const int HasSetsRequiredMembersPopulatedBit = 0x1 << 28;
             private const int IsUnscopedRefBit = 0x1 << 29;
             private const int IsUnscopedRefPopulatedBit = 0x1 << 30;
+            private const int IsInterceptableBit = 0x1 << 31;
+            private const int IsInterceptablePopulatedBit = 0x1 << 32;
```

Comment:
>0x1 << 32 [](http://example.com/codeflow?start=60&length=9)

Is this legal?

Reply:
Yeah. It seems that this expression is equivalent to `0x1 << 0`.

https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/bitwise-and-shift-operators#shift-count-of-the-shift-operators

> For the x << count, x >> count, and x >>> count expressions, the actual shift count depends on the type of x as follows:
> 
> If the type of x is int or uint, the shift count is defined by the low-order five bits of the right-hand operand. That is, the shift count is computed from count & 0x1F (or count & 0b_1_1111).